db/l2db/apiqueries.go

```diff
@@ -69,6 +69,16 @@ func (l2db *L2DB) AddTxAPI(tx *PoolL2TxWrite) error {
 			feeUSD, l2db.maxFeeUSD))
 	}
 
+	if tx.ToBJJ != nil {
+		var exists bool
+		row := l2db.dbRead.QueryRow(`SELECT EXISTS(SELECT 1 FROM account WHERE bjj = $1);`, tx.ToBJJ)
+		if err := row.Scan(&exists); err != nil {
+			return tracerr.Wrap(err)
+		} else if !exists {
```

Comment:
Maybe split this if/else chain in two verifications would it make the code more clean. 
One if verifies the error
Other if, after the value is set to exists variable, check the exists variable content.